hut_10sqft/hut_10sqft/suco_installer.py

```diff
@@ -55,9 +57,13 @@ class SucoInstaller():
     _MSG_CONSOLE_TOOL_INTRO = ""
     _MSG_ARG_PATH_COLCONWS = f"Path to the temporary Colcon workspace for building and installing SUCO itself. \
         If not passed then the path will be the default '{CompInitSetupConfig.PATH_TEMP_COLCON_WS}'."
+    _MSG_PIP_BREAK_SYSPKG = "If specified, install packages by pip even if the package is already installed by \
+        the system package manager (e.g., apt). This may break the system packages."
+
+    _PKG_COLCON_PIP = "colcon-common-extensions colcon-core"
```

Comment:
Very temporary workaround for `colcon-core` not found error https://github.com/kinu-garage/hut_10sqft/pull/1300#discussion_r2491948335
This is not an ideal solution at all.